mteb/tasks/PairClassification/ara/ArEntail.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskPairClassification import AbsTaskPairClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class ArEntail(AbsTaskPairClassification):
+    metadata = TaskMetadata(
+        name="ArEntail",
+        dataset={
+            "path": "arbml/ArEntail",
+            "revision": "4da4316c6e3287746ab74ff67dd252ad128fceff",
+        },
+        description="A manually-curated Arabic natural language inference dataset from news headlines.",
+        reference="https://link.springer.com/article/10.1007/s10579-024-09731-1",
+        type="PairClassification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["ara-Arab"],
+        main_score="ap",
+        date=("2024-05-04", "2024-05-04"),
```

Comment:
The research paper was released in April. So, this would be before that.
Doesn't the paper specify the data collection years?

Reply:
Good point, I updated for: from 2020-01-01 (based on randomly googling sentences from the dataset and finding the news articles) to the acceptance date